<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitView</class>
 <widget class="QWidget" name="GitView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QtWaitingSpinner" name="branchSpinner" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="lbBranch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Branch:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBranch"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="logWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="LogGraph" name="logGraph" native="true"/>
      <widget class="LogView" name="logView" native="true"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QtWaitingSpinner" name="diffSpinner" native="true"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Commit ID:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leSha1"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QtWaitingSpinner" name="findSpinner" native="true"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Find</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>commit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbFindWhat">
             <item>
              <property name="text">
               <string>adding/removing string</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>changing lines matching</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>containing</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leFindWhat">
             <property name="placeholderText">
              <string>Press Enter to find commits</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbFindType">
             <item>
              <property name="text">
               <string>Exact</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ignore case</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Regexp</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbPrev">
             <property name="toolTip">
              <string>Find previous</string>
             </property>
             <property name="text">
              <string>↑</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbNext">
             <property name="toolTip">
              <string>Find next</string>
             </property>
             <property name="text">
              <string>↓</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="DiffView" name="diffView" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DiffView</class>
   <extends>QWidget</extends>
   <header>diffview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogView</class>
   <extends>QWidget</extends>
   <header>logview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtWaitingSpinner</class>
   <extends>QWidget</extends>
   <header>waitingspinnerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogGraph</class>
   <extends>QWidget</extends>
   <header>logview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbBranch</tabstop>
  <tabstop>logWidget</tabstop>
  <tabstop>leSha1</tabstop>
  <tabstop>cbFindWhat</tabstop>
  <tabstop>leFindWhat</tabstop>
  <tabstop>cbFindType</tabstop>
  <tabstop>tbPrev</tabstop>
  <tabstop>tbNext</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
